packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.scss

```diff
@@ -47,6 +47,9 @@
 
             &__balance {
                 margin-left: auto;
+                &-total {
+                    text-align: end;
+                }
```

Comment:
same here

Reply:
`total` isn't modifier. For me, `__balance-total` is an element. But it could be element inside element. Like this: `__balance__total`